src/ImageSharp/Configuration.cs

```diff
@@ -75,6 +75,24 @@ public int MaxDegreeOfParallelism
             }
         }
 
+        /// <summary>
+        /// Gets or sets the size of the buffer to use when working with streams.
+        /// Intitialized with <see cref="DefaultStreamProcessingBufferSize"/> by default.
+        /// </summary>
+        public int StreamProcessingBufferSize
+        {
+            get => this.streamProcessingBufferSize;
+            set
+            {
+                if (value <= 0)
```

Comment:
Buffer size 1 and 2 will not work.

Reply:
I’ll have a look at your tests and fix up

Reply:
We're missing some guard sanitation on the `Position` property but the problem is really with the tests.  They're expecting a minimum length.

Reply:
The main question: would `Image.Load(Async)` work with buffer sizes like `1`, `2`, `3`, `503`, `719` ? If not, the setter should throw.

Reply:
It'll work with any length. Was just bad tests.